server/webdriver_tests/place_explorer_test.py

```diff
@@ -91,7 +91,7 @@ def test_demographics_link(self):
         chart_title = age_across_places_chart.find_element_by_tag_name(
             "h4").text
         self.assertEqual(
-            "Population by Gender across states near California(2018)",
+            "Median Age by Gender across states near California(2018)",
```

Comment:
this will have to be updated post-merge from upstream/master as well

Reply:
done